mobile/App.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import RNLanguages from 'react-native-languages';
+import * as RNLocalize from 'react-native-localize';
```

Comment:
Could you use named imports here?

Reply:
had copypasted from docs, but checked now, can use named imports